### ISSUE TITLE
Feature is only available on certified Alexa built-in devices with web application

### ISSUE THREAD

**Issue by GOGCerence** [alexa/avs-device-sdk#1996]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:

Cerence is building a web application that connects to Alexa using AVS HTTP endpoints directly for a variety of actions on the user home graph. While this has been working well for a while, we recently (two weeks or so) are facing an issue where endpoints always return "Feature is only available on certified Alexa built-in devices". This is in a development environment.   

## What is the expected behavior?

The expected behavior should be what it was before, AVS endpoint executing our HTTP queries. For example, "Turn on the lights in the living room" or "Set the temperature to 20 degrees".  

## What behavior are you observing?

Instead, we receive a notification from Alexa that the "Feature is only available on certified Alexa built-in devices" and no action is executed.

## Provide the steps to reproduce the issue, if applicable:

Since this product is in development and not yet public, it is not reproducible externally. However, any basic commands we want to support triggers this error, such as the examples previously mentioned. It has been suggested that this issue is due to our application not going through the device certification process but since this is a development environment and that we are not exactly building a device but rather a web application, we are not sure how to go forward and resolve this. 

### What version of the AVS Device SDK are you using?
      We are using AVS HTTP endpoints directly.

### Tell us what hardware you're using:
- [X] Desktop / Laptop
- [ ] Raspberry Pi
- [ ] Other - tell us more:

### Tell us about your OS (Type & version):
- [X] Linux
- [ ] MacOS 
- [ ] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

**Comment by smirkhani** [comment]
We also have the same issue. We are also in the development phase now. We were hoping to finish our QC and start the certification process. But, we encountered the same issue.

**Comment by kjkh** [comment]
Hi, this behavior is due to the fact that your device is uncertified; only limited features are available for uncertified devices: https://github.com/alexa/avs-device-sdk/security/policy. Once you have certified your device, it should work as expected. You can find out more about device certification here: https://developer.amazon.com/en-US/alexa/devices/connected-devices/business-resources/works-with-alexa.

If you have an Amazon business representative, you may reach out  to them as well for more information.

**Comment by seyedmaysamlavasani** [comment]
@kjkh - but in order to do the certification we need to first develop the product - please note that this is happening on a product which is under development

**Comment by kjkh** [comment]
@seyedmaysamlavasani 

Yes, we understand this. Do you have an Amazon Business representative you're working with that you can reach out to? If not, you can reach out to avs-security@amazon.com, who may be able to assist further.

**Comment by seyedmaysamlavasani** [comment]
Thanks @kjkh for your message - we don’t have any Amazon business representative - I googled how to contact a representative. Not quite clear to me how to find one. Can you please help us?

Sure - I will send our request to security@amazon.com

**Comment by koshisan** [comment]
I am not quite sure I understand the process: AVS on non-commercial testing devices like a home-build raspberry pi cannot be used to control any smarthome function anymore? You need to go through the whole certification process? 
The documentation does not reflect this! https://github.com/alexa/avs-device-sdk/security/policy talks only about a consent screen during device registration...

**Comment by Eymerich01** [comment]
I was developing a product but suddenly device management changed. This is not correct!

**Comment by azlsfrossard** [comment]
> Thanks @kjkh for your message - we don’t have any Amazon business representative - I googled how to contact a representative. Not quite clear to me how to find one. Can you please help us?
> 
> Sure - I will send our request to [security@amazon.com](mailto:security@amazon.com)

Hi, were you able to find your Amazon business representative?

Also, I noticed you may have used security@amazon.com instead of avs-security@amazon.com.

Finally, you may also be able to get additional information on how to find a business representative using this form:

https://developer.amazon.com/alexa/console/contact-us
